backend/uclapi/dashboard/views.py

```diff
@@ -71,7 +72,8 @@ def dashboard(request):
         user_id = request.session["user_id"]
     except KeyError:
         url = os.environ["SHIBBOLETH_ROOT"] + "/Login?target="
-        param = request.build_absolute_uri(request.path) + "user/login.callback"
+        param = (request.build_absolute_uri(request.path) +
+                 "user/login.callback")
```

Comment:
I think we should decide what URL naming we should use going forward.

The dot syntax is the third different style out of the ones we've used so far haha